src/src/PluginStructs/P131_data_struct.cpp

```diff
@@ -165,35 +165,42 @@ void P131_data_struct::cleanup() {
   matrix = nullptr;
 }
 
+/****************************************************************************
+ * loadContent: load the default content if not yet loaded
+ ***************************************************************************/
+void P131_data_struct::loadContent(struct EventStruct *event) {
+  if (!stringsInitialized) {
+    LoadCustomTaskSettings(event->TaskIndex, strings, P131_Nlines, 0);
+    stringsInitialized = true;
+  }
+
+  if (!contentInitialized && stringsInitialized) {
+    content.clear();
+    content.reserve(P131_CONFIG_TILE_HEIGHT);
+
+    for (uint8_t x = 0; x < P131_CONFIG_TILE_HEIGHT; x++) {
+      content[x] = P131_content_struct();
```

Comment:
This will probably cause a crash, as reserve != resize.
For a std::vector, reserve is just the allocation, resize is actually extending the size of the container.
After a clear, the size is set to 0, so in order to increase the size, you either need to use `push_back` or `resize`.

Reply:
Corrected